src/kOS.Safe/Compilation/KS/kRISC.tpg

```diff
@@ -199,7 +201,8 @@ unset_stmt     -> UNSET (IDENTIFIER | ALL) EOI;
 // ---------- expressions ---------------------------
 
 arglist            -> expr (COMMA expr)*;
-expr               -> and_expr (OR and_expr)*;
+expr               -> (or_expr|instruction_block);
+or_expr            -> and_expr (OR and_expr)*;
```

Comment:
Top level expression is now allowed to also be an instruction block with braces (thus the anonymous function syntax).